### ISSUE TITLE
Latest commit causes index error on Windows

### ISSUE THREAD

**Issue by tarensanders** [tzutalin/labelImg#655]
- **OS:** Windows 10
- **PyQt version:** 5.12.0

The latest commit (31463497c89928e7d2ea47d3e8bf5a953fdf8aeb) does not seem to be compatible with Windows file names:

```
Traceback (most recent call last):
  File "labelImg.py", line 1271, in openDirDialog
    self.importDirImages(targetDirPath)
  File "labelImg.py", line 1282, in importDirImages
    self.openNextImg()
  File "labelImg.py", line 1355, in openNextImg
    self.loadFile(filename)
  File "labelImg.py", line 1096, in loadFile
    self.showBoundingBoxFromAnnotationFile(filePath)
  File "labelImg.py", line 1114, in showBoundingBoxFromAnnotationFile
    filedir = filePath.split(basename)[0].split("/")[-2:-1][0]
IndexError: list index out of range
```

Specifically, ln 1114 in `labelImg.py` is: `filedir = filePath.split(basename)[0].split("/")[-2:-1][0]` which will always fail on Windows.

**Comment by BinhLe2711** [comment]
Open labelImg.py and change line 1114 to
`filedir = filePath.split(basename)[0].split("\\")[-2:-1][0]`

**Comment by shivtejshete** [comment]
update the corresponding folder with following :

 def show_bounding_box_from_annotation_file(self, file_path):
        if self.default_save_dir is not None:
            basename = os.path.basename(os.path.splitext(file_path)[0])
            xml_path = os.path.join(self.default_save_dir, basename + XML_EXT)
            txt_path = os.path.join(self.default_save_dir, basename + TXT_EXT)
            json_path = os.path.join(self.default_save_dir, basename + JSON_EXT)

**Comment by tzutalin** [comment]
Isn’t it fixed? It should not exist

**Comment by tzutalin** [comment]
It was fixed by 62585531ca1a6e7450f1e108e137a3fe67365e68
